parsl/providers/kubernetes/kube.py

```diff
@@ -104,7 +104,10 @@ def __init__(self,
         if not _kubernetes_enabled:
             raise OptionalModuleMissing(['kubernetes'],
                                         "Kubernetes provider requires kubernetes module and config.")
-        config.load_kube_config()
+        try:
+            config.load_kube_config()
+        except:
```

Comment:
This line is caught by flake8: it catches all exceptions.

In practice in parsl, this sort of thing has caused problems such as ctrl-C handling: https://github.com/Parsl/parsl/issues/2174

Maybe it should be more narrowly `kubernetes.config.config_exception.ConfigException`

Reply:
Sounds reasonable.

Reply:
yes it is, try this try block, and the issue persists
```
    try:
        config.load_incluster_config()
    except ConfigException as e:
        logging.exception(e)
        config.load_config()

```